pages/public/contact_us.vue

```diff
@@ -90,12 +101,13 @@ export default {
         name: '',
         email: '',
         message: '',
+        cellphone: '',
       },
       nameRules: [v => !!v || 'Digite seu nome'],
       messageRules: [v => !!v || 'Digite uma mensagem'],
+      cellphoneRules: [v => !!v || 'Digite seu celular com o DDD'],
       emailRules: [
-        v => !!v || 'Digite o e-mail',
-        v => /.+@.+\..+/.test(v) || 'E-mail inválido',
+        // v => /.+@.+\..+/.test(v) || 'E-mail inválido',
```

Comment:
```suggestion
        v => (v === '' || /.+@.+\..+/.test(v)) || 'E-mail inválido',
```
Com esse ajuste se o e-mail não tiver sido preenchido deixa passar, mas se for preenchido obriga a validaçã do formato do e-mail.

Reply:
ajustado